### PR TITLE
Track Screens, Vehicle Details Screens, Fastag Screens Implemented for Web

### PR DESCRIPTION
**Pull request by Vedantfuley**
fastag Screens, Track Screens, Vehicle Details Screens implemented for web
ongoing UI issues are resolved 
-> While adding a new driver if the driver already exists it shows an alert pop-up.
-> While adding a new truck if the truck already exists it shows an alert pop-up.
-> If tracking is not available on a particular device this will show up alert dialog box.

### CODE REVIEW

File: lib/functions/trackScreenFunctions.dart [hunk 1]
```diff
@@ -150,13 +152,13 @@ getStoppageHistory(int? deviceId, String from, String to) async {
...
   var logger = Logger();
-  logger.i("in polyline after function");
+  // logger.i("in polyline after function");
   polylineCoordinates.clear();
   int a = 0;
   int b = a + 3;
   int c = 0;
-  print("length ${gpsDataHistory.length}");
-  print("End lat ${gpsDataHistory[gpsDataHistory.length - 1].latitude}");
+  // print("length ${gpsDataHistory.length}");
```
Comment (Anik-Chakraborty) [inline lib/functions/trackScreenFunctions.dart:160]: remove these

File: lib/functions/trackScreenFunctions.dart [hunk 2]
```diff
@@ -173,31 +175,31 @@ getPoylineCoordinates(var gpsDataHistory, List<LatLng> polylineCoordinates) {
   } // get polyline between every two lat long obtained from response body
 
   if (gpsDataHistory.length % 2 == 0) {
-    print("In even ");
+    // print("In even ");
```
Comment (Anik-Chakraborty) [inline lib/functions/trackScreenFunctions.dart:178]: Also these.. and other places

**Review by Anik-Chakraborty — APPROVED** [review]
Always formate the code

**Comment by Anik-Chakraborty** [comment]
Press ctrl+alt+l at every file to format....

**Review by liveasy97 — APPROVED** [review]

**Pull request merged**